src/libraries/System.Private.CoreLib/src/System/AttributeUsageAttribute.cs

```diff
@@ -18,14 +18,15 @@ namespace System
     public sealed class AttributeUsageAttribute : Attribute
     {
         private readonly AttributeTargets _attributeTarget; // Defaults to all
-        private bool _allowMultiple = false; // Defaults to false
-        private bool _inherited = true; // Defaults to true
+        private bool _allowMultiple;
+        private bool _inherited;
 
         internal static readonly AttributeUsageAttribute Default = new AttributeUsageAttribute(AttributeTargets.All);
```

Comment:
You can also change this to `const` while you are on it to get rid of the static constructor.

Reply:
I cannot as `AttributeUsageAttribute` is a class